tmt/steps/provision/podman.py

```diff
@@ -24,6 +24,10 @@ def __init__(self, data, step):
         self.container_name = None
         self.container_id = None
 
+        # Environment variables for podman command
+        self.podman_env = ' '.join(
+            f"-e '{env}'" for env in self.opt('env_'))
```

Comment:
Based on this all of the provisioners now need to parse these opts, right?

@psss:
Also: `discover` needs to do the proper override, in case the env is defined in L1, right?

Reply:
this feature is now only for podman, but we should in next patches add it to other provisioners yes ....

Reply:
Yes, this will need to be added to `discover` step as well.